### PR TITLE
Add eu.ithz.umftpd

### PR DESCRIPTION
**Pull request by ergoithz**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: eu.ithz.umftpd.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+app-id: eu.ithz.umftpd
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: /app/bin/umftpd
```
Comment (hfiguiere) [inline eu.ithz.umftpd.yaml:5]:
```suggestion
command: umftpd
```

File: flathub.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: this is disappointing.

File: eu.ithz.umftpd.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  - type: file
+    url: https://files.pythonhosted.org/packages/2d/86/3ca57cddfa0419f6a95d1c8478f8f622ba597e3581fd501bbb915b20eb75/cffi-1.15.1-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+    sha256: 5d598b938678ebf3c67377cdd45e09d431369c3b1a5b331058c338e201f12b27
+  - type: file
+    url: https://files.pythonhosted.org/packages/d5/9f/9c0e3288b85f907a008f9d31318b0e4de31b2f67724a8745e633741f609c/pyOpenSSL-22.0.0-py2.py3-none-any.whl
+    sha256: ea252b38c87425b64116f808355e8da644ef9b07e429398bfece610f893ee2e0
+  - type: file
+    url: https://files.pythonhosted.org/packages/31/61/63ef60aca6de07eba1639d9d47f3f8e29462e8bb49d6a8dce9aeff240646/pyftpdlib-1.5.6.tar.gz
+    sha256: fda655d81f29af52885ca2f8a2704134baed540f16d66a0b26e8fdfafd12db5e
+finish-args:
```
Comment (hfiguiere) [inline eu.ithz.umftpd.yaml:45]: finish-args is usually above modules.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build eu.ithz.umftpd

File: flathub.json (hunk 2 above)
Comment (ergoithz) [reply flathub.json:2]: Indeed, I am limited by the wheels provided by upstream projects and flatpak basic builder handling of python packages.

Comment (hfiguiere) [reply flathub.json:2]: I beg to disagree

image

image

Comment (ergoithz) [reply flathub.json:2]: Is this is blocking to get it published on flathub, if so, do you mind linking where it is stated in the documentation?

File: flathub.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (hfiguiere) [inline flathub.json:1]: you can remove that file.

File: eu.ithz.umftpd.yaml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    url: https://files.pythonhosted.org/packages/a4/53/bfc6409447ca024558b8b19d055de94c813c3e32c0296c48a0873a161cf5/setuptools-63.2.0-py3-none-any.whl
+    sha256: 0d33c374d41c7863419fc8f6c10bfe25b7b498aa34164d135c622e52580c6b16
+  - type: file
+    url: https://files.pythonhosted.org/packages/d3/91/4df247dd4da18b72b5bbabe1fa2b85029c34e1d6f0afdd6329d15d6bf2b5/importlib_resources-5.9.0-py3-none-any.whl
+    sha256: f78a8df21a79bcc30cfd400bdc38f314333de7c0fb619763f6b9dabab8268bb7
+  - type: file
+    url: https://files.pythonhosted.org/packages/f0/36/639d6742bcc3ffdce8b85c31d79fcfae7bb04b95f0e5c4c6f8b206a038cc/zipp-3.8.1-py3-none-any.whl
+    sha256: 47c40d7fe183a6f21403a199b3e4192cca5774656965b0a4988ad2f8feb5f009
+  - type: file
+    url: https://files.pythonhosted.org/packages/20/8b/66600f5851ec7893ace9b74445d7eaf3499571b347e339d18c76c876b0f9/cryptography-37.0.4-cp36-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```
Comment (hfiguiere) [inline eu.ithz.umftpd.yaml:40]:
```suggestion
    url: https://files.pythonhosted.org/packages/20/8b/66600f5851ec7893ace9b74445d7eaf3499571b347e339d18c76c876b0f9/cryptography-37.0.4-cp36-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
    only-arches:
      - x86_64
```

File: eu.ithz.umftpd.yaml [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+    sha256: f2dcb0b3b63afb6df7fd94ec6fbddac81b5492513f7b0436210d390c14d46ee8
+  - type: file
+    url: https://files.pythonhosted.org/packages/fd/5a/f47456f062b0c5bd828198992fca1f78bcc7aeadd216d9ce6c3348188b92/cryptography-37.0.4-cp36-abi3-manylinux_2_17_aarch64.manylinux2014_aarch64.whl
```
Comment (hfiguiere) [inline eu.ithz.umftpd.yaml:43]:
```suggestion
    url: https://files.pythonhosted.org/packages/fd/5a/f47456f062b0c5bd828198992fca1f78bcc7aeadd216d9ce6c3348188b92/cryptography-37.0.4-cp36-abi3-manylinux_2_17_aarch64.manylinux2014_aarch64.whl
    only-arches:
      - aarch64
```

File: eu.ithz.umftpd.yaml [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+    sha256: 80f49023dd13ba35f7c34072fa17f604d2f19bf0989f292cedf7ab5770b87a0b
+  - type: file
+    url: https://files.pythonhosted.org/packages/62/d5/5f610ebe421e85889f2e55e33b7f9a6795bd982198517d912eb1c76e1a53/pycparser-2.21-py2.py3-none-any.whl
+    sha256: 8ee45429555515e1f6b185e78100aea234072576aa43ab53aefcae078162fca9
+  - type: file
+    url: https://files.pythonhosted.org/packages/2d/86/3ca57cddfa0419f6a95d1c8478f8f622ba597e3581fd501bbb915b20eb75/cffi-1.15.1-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```
Comment (hfiguiere) [inline eu.ithz.umftpd.yaml:49]:
```suggestion
    url: https://files.pythonhosted.org/packages/2d/86/3ca57cddfa0419f6a95d1c8478f8f622ba597e3581fd501bbb915b20eb75/cffi-1.15.1-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
    only-arches:
      - x86_64
```

File: eu.ithz.umftpd.yaml [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+    sha256: 5d598b938678ebf3c67377cdd45e09d431369c3b1a5b331058c338e201f12b27
+  - type: file
+    url: https://files.pythonhosted.org/packages/77/b7/d3618d612be01e184033eab90006f8ca5b5edafd17bf247439ea4e167d8a/cffi-1.15.1-cp39-cp39-manylinux_2_17_aarch64.manylinux2014_aarch64.whl
```
Comment (hfiguiere) [inline eu.ithz.umftpd.yaml:52]:
```suggestion
    url: https://files.pythonhosted.org/packages/77/b7/d3618d612be01e184033eab90006f8ca5b5edafd17bf247439ea4e167d8a/cffi-1.15.1-cp39-cp39-manylinux_2_17_aarch64.manylinux2014_aarch64.whl
    only-arches:
      - aarch64
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by barthalion** [comment]
bot, build eu.ithz.umftpd

**Pull request closed without merging**